test/runtests.jl

```diff
@@ -13,4 +13,37 @@ w = Window(Blink.@d(:show => false)); sleep(5.0)
 
 @test sprint(Blink.jsexpr, :(Dict("a" => 1, :b => 10))) == "{\"a\":1,b:10}"
 
+@testset "content! Tests" begin
```

Comment:
Moving this testset to `test/api.jl` would be great :)

Reply:
Oops! Thanks. I started rebasing this, but never finished it looks like.. Let me push that up now!